src/resolve/resolved-manager.h

```diff
@@ -31,6 +31,14 @@ typedef struct EtcHosts {
         Set *no_address;
 } EtcHosts;
 
+typedef struct DNSStubListenerExtra {
+        int fd;
+
+        DnsStubListenerMode mode;
+        SocketAddress address;
+        sd_event_source *dns_stub_extra_event_source;
```

Comment:
Not a matter, but how about simply naming `event_source`?

Reply:
bitfield are used to save data in network mostly. Host system is not required.